src/components/UserProfile/UserLinks/UserLinks.jsx

```diff
@@ -11,7 +10,7 @@ const UserLinks = ({
         if (item.Link.includes('http')) {
           return (
             <React.Fragment key={item.Name}>
-              <a key={item.link} href={item.Link} target="_blank">
+              <a key={item.link} href={item.Link} target="_blank" rel="noreferrer">
```

Comment:
The key attribute of the <a> tag is using item.link, but there is no link property in the item object.  I think item.Link should be used as the value for the key attribute.